### PR TITLE
Update projects to support Microbuild v2

### PR DESCRIPTION
**Pull request by pieandcakes**
@jacdavis @gregg-miskelly @wiktork @chuckries 

Adding lines to support MicroBuild v2

### CODE REVIEW

File: src/DebugEngineHost.Stub/project.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "dnxcore50.app": {}
   },
   "dependencies": {
+    "MicroBuild.Core": "0.2.0",
```
Comment (wiktork) [inline src/DebugEngineHost.Stub/project.json]: You'll also need FilesToSign in each proj file.

**Comment by gregg-miskelly** [comment]
:+1:

**Comment by chuckries** [comment]
:+1:

File: src/DebugEngineHost.Stub/project.json (hunk 1 above)
Comment (pieandcakes) [inline src/DebugEngineHost.Stub/project.json]: @wiktork Are you talking about in the project.json or in the .proj file?

Comment (wiktork) [inline src/DebugEngineHost.Stub/project.json]: So it looks like FilesToSign is already part of DropFiles.targets, (which is included in MIDebugPackage.csproj). However, the final binaries do not appear to be signed.

Comment (gregg-miskelly) [inline src/DebugEngineHost.Stub/project.json]: My guess: This adds MicroBuild.Core dependency to three projects that probably don't actually need the dependency at all, and doesn't add it to the project that actually needs the dependency (MIDebugPackage)

**Comment by pieandcakes** [comment]
I will follow up with Fangliang and Wiktor. Cancelling this pull request for now.

**Pull request closed without merging**